.github/workflows/pypi.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Publish to PyPI
-        uses: uclahs-cds/tool-pypi-publish-action/publish-to-pypi@0d60e0c0d46ea8d0486bb15d8940587121f44fd2
+        uses: uclahs-cds/tool-pypi-publish-action/publish-to-pypi@58e64514405eafe4f8c40130001cbd4d62acdf5c
         with:
           package-name: mopepgen
+          url: https://upload.pypi.org/legacy/
```

Comment:
You may want to consider using a workflow dispatch variable here as well, so you can trigger a test upload through this repository if it's ever needed.

Otherwise this looks good to me.

Reply:
Yeah that's a good idea. I'll figure it out when I need this.